example/src/main/java/com/klarna/example/MainActivity.java

```diff
@@ -74,6 +78,17 @@ public void openKlarnaPreferences(View view) {
 
     }
 
+    public void onBuy(View view) {
```

Comment:
If it's the same dialog over and over again - Why not have a member and just set when null, and show when needed. 
but make sure to dispose properly on activity lifecycle